docs/package.json

```diff
@@ -16,9 +16,11 @@
     "lint:js": "eslint --ignore-path .gitignore --fix .",
     "lint:css": "stylelint \"src/**/*.css\"",
     "lint": "npm-run-all --parallel lint:*",
+    "rebuild-phenomic": "cd ../ && npm -s run transpile",
```

Comment:
Do you think this is the best approach? 

I usually left a terminal tab open which is running this command: 

```
npm run transpile -- --watch
```

Combining this with `npm-run-all` in `start`, we won't have to kill Webpack process to rebuild Phenomic.